addons/account/models/account.py

```diff
@@ -1402,6 +1401,13 @@ class AccountTaxGroup(models.Model):
     property_tax_receivable_account_id = fields.Many2one('account.account', company_dependent=True, string='Tax current account (receivable)')
     property_advance_tax_payment_account_id = fields.Many2one('account.account', company_dependent=True, string='Advance Tax payment account')
 
+    def _any_is_configured(self):
+        domain = expression.OR([[('property_tax_payable_account_id', '!=', False)],
+                                [('property_tax_receivable_account_id', '!=', False)],
+                                [('property_advance_tax_payment_account_id', '!=', False)]])
+        group_with_config = self.search(domain, limit=1)
+        return len(group_with_config) > 0
```

Comment:
```
return self.search_count([
  '|', '|',
  ('property_tax_payable_account_id', '!=', False),
  ('property_tax_receivable_account_id', '!=', False),
  ('property_advance_tax_payment_account_id', '!=', False),
])
```
It seems you miss a filter about the company?

Reply:
Done

Reply:
replace by `search_count` plz :)